api/src/test/java/org/openmrs/DurationTest.java

```diff
@@ -98,7 +99,8 @@ public void addToDate_shouldFailWhenUnitIsRecurringAndFrequencyIsUnknown() throw
 		OrderFrequency frequency = null;
 		
 		expectedException.expect(APIException.class);
-		expectedException.expectMessage(Matchers.is("Duration.error.frequency.null"));
+		expectedException.expectMessage(Matchers.is(Context.getMessageSourceService().getMessage(
```

Comment:
@wluyima I'm not sure what do you mean?
Tests work only when you retrieve value from .properties files using Context.getMessageSourceService().getMessage(
or put it directy instead of key